libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -1,5 +1,6 @@
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNetCore.Http;
+using TNO.API.Areas.Editor.Models.Content;
```

Comment:
Do not do this.  Most endpoints will have their own models to stop collusion and regression issues when we need to make changes to one.  Create a `PorductModel` for this namespace `TNO.API.Areas.Services.Models.Content`.

Reply:
Updated, thanks